stock_inventory_discrepancy/views/stock_warehouse_view.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017-2020 ForgeFlow S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="view_warehouse_form" model="ir.ui.view">
```

Comment:
Had to look it up in the contributing guidelines but it is preferred "naming the inherited view after the original over using a name which follows the guidelines. This eases looking up the original view and other inheritance if they all have the same name."
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#253inherited-xml